_posts/2024-06-30-concurrent-search-follow-up.md

```diff
@@ -0,0 +1,307 @@
+---
+layout: post
+title:  "Exploring concurrent segment search performance"
+authors:
+ - jaydeng
+ - sohami
+date: 2024-06-30
```

Comment:
If you are able to get all the updates done today, we can push it live this afternoon. Please update the date to today if you feel this is possible.